benchmarks/benchmark_bitpacking.py

```diff
@@ -21,207 +20,74 @@ def benchmark(function, num_runs, setup =None):
 
 
 def test_vs_existing():
-    def new_():
-        fake_tensor = torch.randint(0, 2**8, (1, 1024,1024), dtype=torch.uint8).cuda()
+    def new_(scale):
+        fake_tensor = torch.randint(2**8-1, (1, scale,scale), dtype=torch.uint8).cuda()
```

Comment:
small detail: -1 is not needed since `torch.randint()` only samples from `[low,high)`